cmds/core/lock-msrs/lock-msrs.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// lock-msrs locks a bunch of intel MSRs, all specified in the Intel
+// Software developer's manual. This seems like a good set of bits to
+// lock down when booting through NERF/LINUXBOOT to some other OS.
+package main
+
+import (
+	"log"
+
+	"github.com/u-root/u-root/pkg/msr"
+)
+
+const usage = `msr r glob register
+msr w glob register value`
+
+const MSR_IA32_FEATURE_CONTROL   uint32 = 0x3A
+const MSR_PKG_CST_CONFIG_CONTROL uint32 = 0xE2
+const MSR_FEATURE_CONFIG         uint32 = 0x13C
+const MSR_DRAM_POWER_LIMIT       uint32 = 0x618
+const MSR_CONFIG_TDP_CONTROL     uint32 = 0x64B
+const IA32_DEBUG_INTERFACE       uint32 = 0xC80
```

Comment:
Can you document briefly what each of them does when "locked"?

Reply:
I'll leave a comment, but locking the msrs just mean they can't be changed until reset.

Reply:
That's a good thing to comment on, but also say what they each mean when locked? What's IA32_DEBUG_INTERFACE?

Reply:
Added more info.